manta-js/package/src/pallets/MantaSbtWallet.ts

```diff
@@ -84,11 +85,46 @@ export default class MantaSbtWallet
     return result;
   }
 
+  async getTransactionDatas(
+    posts: TransactionPost[],
+  ): Promise<TransactionData[]> {
+    const result = await this.baseWallet.wrapWalletIsBusy(
+      async () => {
+        const transactionDatas = [];
+        for (let i = 0; i < posts.length; i += 1) {
+          const postBody = posts[i];
+          const formatData = new this.wasm.TransferPost(
+            null,
```

Comment:
The `null` is fine for ToPrivate case. but later on when we have requirement for other transfer, then this need to change.

Reply:
right